superset/views/core.py

```diff
@@ -1644,9 +1701,51 @@ def created_dashboards(self, user_id):
 
     @api
     @has_access_api
+    @expose('/user_slices', methods=['GET'])
+    @expose('/user_slices/<user_id>/', methods=['GET'])
+    def user_slices(self, user_id=None):
+        """List of slices a user created, or faved"""
+        if not user_id:
+            user_id = g.user.id
+        Slice = models.Slice  # noqa
+        FavStar = models.FavStar # noqa
+        qry = (
+            db.session.query(Slice,
+                             FavStar.dttm).join(
+                models.FavStar,
+                sqla.and_(
+                    models.FavStar.user_id == int(user_id),
```

Comment:
I feel like we should use the `owners`  field that allows for many owners per slices

Reply:
But then someone needs to ask the owner to be added to the list of owners. This way they can just star a slice in order to use it for an overlay.

Reply:
I'm ok with starring, though we should honor the `owners` field over the `created_by`.